mw-webapp/src/component/footer/Footer.tsx

```diff
@@ -54,27 +51,44 @@ export const Footer = (props: FooterProps) => {
             level={HeadingLevel.h4}
             text={LanguageService.home.socialMedia[props.language]}
             placeholder=""
+            classNameHeading={styles.socialMediaText}
```

Comment:
title by default has weight 500? whe need to additional classname?

Reply:
By default it's 700, but I need 500 in this place (as in the design).
<img src="https://github.com/tritonJS826/masters-way/assets/8752900/f1bc3ae2-97aa-4757-89a0-8d8161b46b66" width="500" />